renpy/loadsave.py

```diff
@@ -414,7 +414,9 @@ def save(slotname, extra_info='', mutate_flag=False):
         if bad is None:
             reraise(t, e, tb)
 
-        e.args = (e.args[0] + ' (perhaps {})'.format(bad),) + e.args[1:]
+        if len(e.args) > 1:
```

Comment:
Think the condition here should be `if len(e.args) > 0:` or even just `if len(e.args):` . As it is we lose the error augmentation in the event that the tuple only has a single item.
The `[1:]` slice is safe on both empty and single item lists, so it's only the direct `[0]` access that needs the guard.
```py
>>> [1][1:]
[]
>>> [][1:]
[]
```